version.php

```diff
@@ -24,8 +24,8 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version   = 2021100103;
-$plugin->release   = '2021100103';
+$plugin->version   = 2023032100;
```

Comment:
Hi @glepod - looks like you need to take a look at the version bump rules :-) - in the latest development branch you bump the full version date, but in older stable branches you only bump by one. more info here:
https://moodledev.io/docs/apis/commonfiles/version.php#version

the rest seems ok to me though.

Reply:
Thanks for the pickup - I missed that :)